packages/block-library/src/media-text/style.scss

```diff
@@ -1,7 +1,12 @@
 .wp-block-media-text {
+	// This block's direction should not be manipulated by rtl, as the mediaPosition control does.
+	/*!rtl:begin:ignore*/
+	direction: ltr;
```

Comment:
Unfortunately, this makes it impossible to have rtl paragraphs nested inside the media text.
Should we add direction ltr (not ignored) to the container of the nested blocks?

Reply:
Ah you're right. It seems to work on the front of site, but not within the editor. You mean the actual `.wp-block-media-text__content` container, correct?

Reply:
I've just pushed an update which adds `direction: ltr` to the `.wp-block-media-text__content` container, ensuring the content containers still abide by proper rtl support. 

Here's a screenshot with a rtl language applied:
<img width="1229" alt="Screen Shot 2019-12-03 at 8 44 13 AM" src="https://user-images.githubusercontent.com/1813435/70056418-6cf6ee80-15a9-11ea-991c-8b01d4663669.png">